### ISSUE TITLE
pip installation failure on Windows

### ISSUE THREAD

**Issue by pepoluan** [escherba/python-metrohash#3]
From within *PyCharm*, I tried installing metrohash, and got the following error:

```
Collecting metrohash
  Downloading metrohash-0.0.13.tar.gz
Building wheels for collected packages: metrohash
  Running setup.py bdist_wheel for metrohash: started
  Running setup.py bdist_wheel for metrohash: finished with status 'error'
  Complete output from command D:\Repos\@VEnv\base-pypy2-01\bin\pypy.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\pepoluan\\AppData\\Local\\Temp\\pycharm-packaging\\metrohash\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d c:\users\pepoluan\appdata\local\temp\tmpqsporapip-wheel- --python-tag pp257:
  running bdist_wheel
  running build
  running build_ext
  building 'metrohash' extension
  creating build
  creating build\temp.win32-2.7
  creating build\temp.win32-2.7\Release
  creating build\temp.win32-2.7\Release\src
  C:\Users\pepoluan\AppData\Local\Programs\Common\Microsoft\Visual C++ for Python\9.0\VC\Bin\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -Iinclude -ID:\Repos\@VEnv\base-pypy2-01\include /Tpsrc/metrohash64.cc /Fobuild\temp.win32-2.7\Release\src/metrohash64.obj -O3 -msse4.2 -Wno-unused-value -Wno-unused-function
  cl : Command line error D8021 : invalid numeric argument '/Wno-unused-value'
  error: command 'C:\\Users\\pepoluan\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\cl.exe' failed with exit status 2
  
  ----------------------------------------
  Running setup.py clean for metrohash
Failed to build metrohash
Installing collected packages: metrohash
  Running setup.py install for metrohash: started
    Running setup.py install for metrohash: finished with status 'error'
    Complete output from command D:\Repos\@VEnv\base-pypy2-01\bin\pypy.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\pepoluan\\AppData\\Local\\Temp\\pycharm-packaging\\metrohash\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record c:\users\pepoluan\appdata\local\temp\pip-esvcxi-record\install-record.txt --single-version-externally-managed --compile --install-headers D:\Repos\@VEnv\base-pypy2-01\include\site\python2.7\metrohash:
    running install
    running build
    running build_ext
    building 'metrohash' extension
    creating build
    creating build\temp.win32-2.7
    creating build\temp.win32-2.7\Release
    creating build\temp.win32-2.7\Release\src
    C:\Users\pepoluan\AppData\Local\Programs\Common\Microsoft\Visual C++ for Python\9.0\VC\Bin\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -Iinclude -ID:\Repos\@VEnv\base-pypy2-01\include /Tpsrc/metrohash64.cc /Fobuild\temp.win32-2.7\Release\src/metrohash64.obj -O3 -msse4.2 -Wno-unused-value -Wno-unused-function
    cl : Command line error D8021 : invalid numeric argument '/Wno-unused-value'
    error: command 'C:\\Users\\pepoluan\\AppData\\Local\\Programs\\Common\\Microsoft\\Visual C++ for Python\\9.0\\VC\\Bin\\cl.exe' failed with exit status 2
    
    ----------------------------------------

  Failed building wheel for metrohash
Command "D:\Repos\@VEnv\base-pypy2-01\bin\pypy.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\pepoluan\\AppData\\Local\\Temp\\pycharm-packaging\\metrohash\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record c:\users\pepoluan\appdata\local\temp\pip-esvcxi-record\install-record.txt --single-version-externally-managed --compile --install-headers D:\Repos\@VEnv\base-pypy2-01\include\site\python2.7\metrohash" failed with error code 1 in C:\Users\pepoluan\AppData\Local\Temp\pycharm-packaging\metrohash\
```

The key seems to be this line:

    cl : Command line error D8021 : invalid numeric argument '/Wno-unused-value'

If you need further info, feel free to ask.

**Comment by Dobatymo** [comment]
Same here (Windows x64, Python 2.7 x64)
```cl : Command line error D8021 : invalid numeric argument '/Wno-unused-value'```

**Comment by TACIXAT** [comment]
I'm getting this on Windows.
```
    Complete output (11 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\user\AppData\Local\Temp\pip-install-hfta02m0\metrohash\setup.py", line 104, in <module>
        long_description=resource_string(__name__, 'README.rst'),
      File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pkg_resources\__init__.py", line 1156, in resource_string
        return get_provider(package_or_requirement).get_resource_string(
      File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pkg_resources\__init__.py", line 1401, in get_resource_string
        return self._get(self._fn(self.module_path, resource_name))
      File "c:\users\user\appdata\local\programs\python\python38\lib\site-packages\pkg_resources\__init__.py", line 1570, in _get
        raise NotImplementedError(
      NotImplementedError: Can't perform this operation for loaders without 'get_data()'` on Windows.
```

**Comment by escherba** [comment]
addressed in #12
